contracts/contracts/token/WOETH.sol

```diff
@@ -66,6 +86,12 @@ contract WOETH is ERC4626, Governable, Initializable {
          * environment.
          */
         oethCreditsHighres = _getOETHCredits();
+        creditsPerTokenLimit = OETH(asset())
```

Comment:
Rounding toward higher yield limits is the best choice here, otherwise the contract would stop allowing any yield at some point.